CHANGELOG.md

```diff
@@ -663,7 +663,7 @@ Note: A subset of crates received patch releases in the 1.2 stream.
     - Allow inversion lists to be built from ranges that include `char::MAX` (#3203)
   - `icu_datetime`: No other changes
   - `icu_decimal`
-    - Add `From<GroupingStrategy>` for `FixedDecimalFormatterOptions` (#3045)
```

Comment:
revert

Reply:
done. I searched for `FixedDecimalFormatterOptions` and I did not find it, but it seems this part of the logs, it can be updated later

Reply:
this is the changelog, at the time when this was added, that type existed. let's not rewrite history